modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/io/exporter/DDMFormInstanceRecordExporterImpl.java

```diff
@@ -185,8 +188,6 @@ protected String getDDMFormFieldValue(
 			List<DDMFormInstanceRecord> ddmFormInstanceRecords, Locale locale)
 		throws Exception {
 
-		DateTimeFormatter dateTimeFormatter = getDateTimeFormatter(locale);
```

Comment:
Hey @antonio-ortega, can you please also remove this method and its references on tests? After your fix, it became useless as the method `formatDate`.

Thanks.